### PR TITLE
Clean up periodic_argument

### PR DESCRIPTION
**Pull request by arihantparsoya**
this is fix for #11166 . 
- The function `unbranched_argument()` is made independent of `periodic_argument()`.
- Docstring need to be added.

### CODE REVIEW

**Comment by parsoyaarihant** [comment]
@asmeurer , can you review this?

File: sympy/functions/elementary/complexes.py [hunk 1]
```diff
@@ -911,9 +909,29 @@ def _eval_evalf(self, prec):
         return (ub - ceiling(ub/period - S(1)/2)*period)._eval_evalf(prec)
 
 
-def unbranched_argument(arg):
-    from sympy import oo
-    return periodic_argument(arg, oo)
+def unbranched_argument(ar):
+    from sympy import exp_polar, log, polar_lift, oo
```
Comment (jksuom) [inline sympy/functions/elementary/complexes.py]: The imports should be at the top of the file, if there are no import cycles. `exp_polar` is there already. `polar_lift` is in this file.

**Comment by jksuom** [comment]
`unbranched_argument` looks very much like `_getunbranched`. Would it be possible to include only one function?

**Comment by asmeurer** [comment]
`unbranched_argument` should be a Function subclass. The whole point is to make it a separate function from the user point of view.

File: sympy/functions/elementary/complexes.py [hunk 2]
```diff
@@ -911,9 +909,29 @@ def _eval_evalf(self, prec):
...
+    unbranched = 0
+    for a in args:
+        if not a.is_polar:
+            unbranched += arg(a)
+        elif a.func is exp_polar:
+            unbranched += a.exp.as_real_imag()[1]
+        elif a.is_Pow:
+            re, im = a.exp.as_real_imag()
+            unbranched += re*unbranched_argument(
+                a.base) + im*log(abs(a.base))
```
Comment (asmeurer) [inline sympy/functions/elementary/complexes.py]: Is this new behavior?

**Pull request closed without merging**